TA-microsoft-graph-security-score/README.md

```diff
@@ -10,11 +10,11 @@ The Microsoft Graph Security Score Add-on for Splunk allows users to collect the
 
 
 * Author - CrossRealms International Inc.
-* Version - 1.0.1
-* Build - 1
+* Version - 1.1.0
+* Build - 2
```

Comment:
Remove Version and Build line from README.

Reply:
okay, are we incrementing the build number in the app.conf while generating the build?

Reply:
No need to change that. it's okay

Reply:
Done